MobileCenter/MobileCenter/Internals/Channel/MSChannel.h

```diff
@@ -71,6 +73,23 @@ typedef void (^enqueueCompletionBlock)(BOOL);
  */
 - (void)removeDelegate:(id<MSChannelDelegate>)delegate;
 
+/**
+ * Notify the caller that this channel is done flushing logs by executing the given block.
+ *
+ * @param completion Completion block to be called when done flushing logs.
+ */
+- (void)notifyWhenDoneFlushingWithCompletion:(MSDoneFlushingCompletionBlock)completion;
+
+/**
+ * Cancel notifiying when done flushing;
+ */
+- (void)cancelNotifyingWhenDoneFlushing;
+
+/**
+ * Resume the channel, it resume normal activity. A channel can't be resumed if it's disabled or the sender is still supsended.
```

Comment:
The beginning is a little wonky. Maybe "Resume the channel. It will resume it's normal activity."?